book/src/framework/getting_started.md

```diff
@@ -1,61 +0,0 @@
-# 🚀 Getting started
-
-## Prerequisites
-- `Docker` [OrbStack](https://orbstack.dev/) or [Docker Desktop](https://www.docker.com/products/docker-desktop/), we recommend OrbStack (faster, smaller memory footprint)
-- [Golang](https://go.dev/doc/install)
-
-## Test setup
-
-To start writing tests create a directory for your project with `go.mod` and add a package
-```
-go get github.com/smartcontractkit/chainlink-testing-framework/framework
-```
-
-Download our [CLI](https://github.com/smartcontractkit/chainlink-testing-framework/releases/tag/framework%2Fv0.1.8)
```

Comment:
It points to 0.18, while 0.20 is already there, you told me that it will point to the latest build. If it's hard to do - you can change link to https://github.com/smartcontractkit/chainlink-testing-framework/releases